site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -113,172 +121,184 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
   );
 
   return (
-    <FullPageHorizontalForm title="New workspace" onCancel={onCancel}>
-      <HorizontalForm onSubmit={form.handleSubmit}>
-        {Boolean(error) && <ErrorAlert error={error} />}
-        {/* General info */}
-        <FormSection
-          title="General"
-          description="The template and name of your new workspace."
-        >
-          <FormFields>
-            <SelectedTemplate template={template} />
-            {versionId && versionId !== template.active_version_id && (
-              <Stack spacing={1} className={styles.hasDescription}>
-                <TextField
-                  disabled
-                  fullWidth
-                  value={versionId}
-                  label="Version ID"
-                />
-                <span className={styles.description}>
-                  This parameter has been preset, and cannot be modified.
-                </span>
-              </Stack>
-            )}
-            <TextField
-              {...getFieldHelpers("name")}
-              disabled={form.isSubmitting}
-              onChange={onChangeTrimmed(form)}
-              autoFocus
-              fullWidth
-              label="Workspace Name"
-            />
-          </FormFields>
-        </FormSection>
+    <>
+      {mode === "duplicate" && <DuplicateWarningMessage />}
```

Comment:
The only thing that changed for the return value is that the component now returns a React Fragment, so that the `DuplicateWarningMessage` component can conditionally render. Everything else is a change in indentation level